Vector.java

```diff
@@ -0,0 +1,41 @@
+import java.io.*;
+import java.util.*;
+import java.lang.Math; 
+
+public class Vector {
+	
+	List<Double> numbers;
+		
+	Vector (List<Double> numbers) {
+			
+		numbers = numbers;
+			
+	}
+		
+	public double distance(Vector vector2) { // returns distance between two vectors
```

Comment:
nit: usually in methods like this that compare another object to "this" object, the name of the argument is `other`, rather than `vector2`.